src/commons/types.ts

```diff
@@ -21,6 +21,7 @@ export interface Theme {
     dot?: object;
     marking?: object;
     'calendar-list'?: {main?: object};
```

Comment:
Why 'calendar-list' is in a string here? Can't we fix it in the calendar-list style file?

Reply:
thats not my change. Its Lidor commit https://github.com/wix/react-native-calendars/commit/785b07da836a93bd4d2183a82cb47153d0c8bd08

Reply:
Ok. I'll take this